frontend/components/header/HeaderWebsite.vue

```diff
@@ -10,7 +10,7 @@
     }"
   >
     <!-- MARK: Mobile Header -->
-    <div id="mobile-header" class="flex px-4 py-3 md:hidden">
+    <div v-if="!aboveMediumBP" class="flex px-4 py-3">
```

Comment:
removing this id breaks some tests

Reply:
This one's on me, @aasimsyed! Sorry about that!